contentcuration/contentcuration/viewsets/channel.py

```diff
@@ -135,19 +243,16 @@ class ChannelViewSet(ValuesViewset):
 
     def get_queryset(self):
         queryset = Channel.objects.filter(deleted=False).filter(
-            id__in=Channel.objects.filter(
```

Comment:
Why was this removed? Doing the query directly adds `GROUP_BY` clauses to the resultant DB query which heavily impacts query performance.